app/src/test/java/edu/outside2154/gamesense/ModelTest.kt

```diff
@@ -7,6 +7,7 @@ import org.junit.Test
 
 const val EPS = 1.0e-008
 
+/*
```

Comment:
If we're going to comment out a test, we should probably leave a `TODO` and an explanation of some sort.